episodes/12-citation.md

```diff
@@ -52,10 +52,19 @@ There is also an [`@software{...`](https://www.google.com/search?q=git+citation+
 no "umbrella" citation like a paper or book exists for the project you want to
 make citable.
 
+Finally you may wish to include citation information in a [Citation File Format
```

Comment:
I find this a little vague. "you may wish" why exactly? What is the difference compared to CITATION.txt?